### PR TITLE
adding header in between transcriptions

### LINKED ISSUE #51: Organization of multiple transcriptions within a post/comment
**Describe the bug**
If a single post has multiple image links within, the bot will reply with all the transcriptions grouped together.

**To Reproduce**
Call the bot to a post/comment with multiple image links. The reply will be one big response rather than having some sort of separation between each transcription.

**Expected behavior**
To be able to recognize the transcriptions for each image within the single post/comment. Possibly with a header or sufficient spacing

**Screenshots/Logs**
image
Above is each images transcription

image
Above is how the bot was replying in M3

**Project information**
This issue has been present since M3

### PR DESCRIPTION
**Pull request by calebeda**
**transcribeImages** function will now append a header before appending the transcribed text to the transcription list for each URL found

Closes #51

### CODE REVIEW

**Review by baconbm — APPROVED** [review]
Looks good to me, thanks Calebe.

**Comment by calebeda** [comment]
In commit 381df54 I changed the image header to just a line. I thought that this looked much cleaner in the posts

**Review by baconbm — APPROVED** [review]
Agreed @calebeda I think your changes look better than it did previously.

File: src/TextifyBot.py [hunk 1]
```diff
@@ -96,6 +96,8 @@ def improveImage(imagePath):
 
 def transcribeImages(imagesToDL): # download and transcribe a list of image URLs
     transcribedText = []
+    transcriptionSpacing = "\n  ----------------------------  \n"
```
Comment (afig) [inline src/TextifyBot.py]:
```suggestion
    transcriptionSpacing = "\n---\n"
```

In Markdown, three or more `-`'s are converted into a horizontal rule, so no need to have any more than three.

**Review by afig — COMMENTED** [review]
Overall looks good. I agree that a horizontal line looks much better, but see my comment about unnecessary extra `-`'s.

**Review by afig — APPROVED** [review]
Looks good. Thanks @calebeda

**Pull request merged**